lib/Dropdown/DropdownContent.js

```diff
@@ -100,20 +100,18 @@ export function DropdownContent({
     }
   }, [showContent, !!contentElementRef.current]);
 
-  const classNames = cx(
-    `dropdown__content border border-solid border-neutral-90 rounded shadow ${className}`,
-    {
-      'is-active': showContent,
-      'dropdown__content--collapse': collapse,
-      'dropdown__content--right': right,
-      'dropdown__content--centre': centre,
-      'dropdown__content--top': top,
-      'dropdown__content--fix-right': fixRight,
-      'dropdown__content--full': full,
-      'dropdown__content--noborder': noBorder,
-      'dropdown__content--no-transform': noTransform
-    }
-  );
+  const classNames = cx(`dropdown__content shadow ${className}`, {
+    'is-active': showContent,
+    'dropdown__content--collapse': collapse,
+    'dropdown__content--right': right,
+    'dropdown__content--centre': centre,
+    'dropdown__content--top': top,
+    'dropdown__content--fix-right': fixRight,
+    'dropdown__content--full': full,
+    'dropdown__content--noborder': noBorder,
+    'border border-solid border-neutral-90 rounded': !noBorder,
```

Comment:
This must have been a recent fix. We had added some border tailwind classes but they always apply even when `noBorder` is set.